packages/block-library/src/navigation/index.php

```diff
@@ -248,13 +248,93 @@ function block_core_navigation_render_submenu_icon() {
 	return '<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 12 12" fill="none" aria-hidden="true" focusable="false"><path d="M1.50002 4L6.00002 8L10.5 4" stroke-width="1.5"></path></svg>';
 }
 
+/**
+ * Get the classic navigation menu to use as a fallback.
+ *
+ * @return object WP_Term The classic navigation.
+ */
+function block_core_navigation_get_classic_menu_fallback() {
+	$classic_nav_menus = wp_get_nav_menus();
+
+	// If menus exist.
+	if ( $classic_nav_menus && ! is_wp_error( $classic_nav_menus ) && count( $classic_nav_menus ) === 1 ) {
+		// If there's only one classic menu then use it.
```

Comment:
```suggestion
		// Use the first classic menu only. Handles simple use case where user has a single
		// classic menu and switches to a block theme. In future this maybe expanded to
		// determine the most appropriate classic menu to be used based on location.
```

Just wondering whether we should add more context here.